src/WebJobs.Script.WebHost/Models/HostStatus.cs

```diff
@@ -2,16 +2,41 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System.Collections.ObjectModel;
+using System.Reflection;
 using Newtonsoft.Json;
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
 {
     public class HostStatus
     {
+        private static string _assemblyFileVersion;
+
+        public HostStatus()
```

Comment:
Made it a public static readonly field in my latest commit -- see if you like that any better :-)